### PR TITLE
Fixes qdel_null issue

### PR DESCRIPTION
**Pull request by DominikPanic**
Fixes: #2412

### CODE REVIEW

File: code/modules/organs/limbs.dm [hunk 1]
```diff
@@ -937,7 +937,8 @@ Note that amputating the affected organ does in fact remove the infection from t
 			spark_system.set_up(5, 0, owner)
 			spark_system.attach(owner)
 			spark_system.start()
-			QDEL_NULL_IN(src, spark_system, 1 SECONDS)
+			spawn(1 SECONDS)
```
Comment (Rohesie) [inline code/modules/organs/limbs.dm]: `INVOKE_NEXT_TICK` instead, to a proc that does the QDEL_NULL.

Comment (Rohesie) [reply code/modules/organs/limbs.dm]: Actually just a callback.

Comment (DominikPanic) [reply code/modules/organs/limbs.dm]: It can't be a callback though, that's the entire reason I'm making this PR

**Comment by Rohesie** [comment]
`error: undefined field: "spark_system" on /datum/limb`

**Comment by DominikPanic** [comment]
> 
> 
> `error: undefined field: "spark_system" on /datum/limb`

I have no idea how to fix this.

**Comment by spookydonut** [comment]
See #2395 for the fix

**Pull request closed without merging**